### ISSUE TITLE
Collection of executions for Hystrix

### ISSUE THREAD

**Issue by ahus1** [micrometer-metrics/micrometer#503]
In #240 counters were implemented for Hystrix. Part of the implementation was taken for a library I maintain.

I found the previous counter implementation difficult to use, therefore it changed in a recent release: https://github.com/ahus1/prometheus-hystrix/issues/19

TL;DR: The total and error counters are dropped and a _terminal_ tag is added to each event type. This allows for more meaningful aggregations than the previous setup.

Feedback welcome. I'm happy to submit a PR. 

cc: @jkschneider, @checketts, @fiunchinho

**Comment by spencergibb** [comment]
:+1:

**Comment by checketts** [comment]
Yes I definitely want the counters to be as easy to use as possible. I tried to break apart the events to match how you describe them, but I've had very little experience with Hystrix, so I would love any improvements.

I don't love the `terminal` tag, if we could find a way to represent it as its own metric that represents terminal events, that would be simplest and most ideal.

**Comment by jkschneider** [comment]
PR very welcome. Thanks for pointing this out @ahus1

**Comment by ahus1** [comment]
I've submitted a PR. Comments are welcome.
